mu/views/chrome.py

```diff
@@ -0,0 +1,213 @@
+from PyQt5.QtCore import QSize, Qt, pyqtSignal
```

Comment:
`chrome.py` doesn't feel like a great name to me -- it's too jargonny.  Could it be `window.py` instead?